manager/src/main/java/com/redhat/service/bridge/manager/workers/id/WorkerIdProvider.java

```diff
@@ -4,13 +4,33 @@
 
 import javax.enterprise.inject.Produces;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class WorkerIdProvider {
```

Comment:
@lucamolteni Can we add a unit test for `WorkerIdProvider` please now that we have logic in it 👍 
cheers,

Rob

Reply:
Testing with environment variable is complicated. I could abstract the sysenv part and test it WDYT?

Reply:
@lucamolteni Or we could simplify the implementation to look like this:

```
@ApplicationScoped
public class WorkerIdProvider {

    @ConfigProperty(name = "event-bridge.resources.worker.id")
    private String workerId;

    public String getWorkerId() {
        return workerId;
    }
}
```

With the following in the `application.properties`:

```
event-bridge.resources.worker.id=${EVENT_BRIDGE_RESOURCES_WORKER_ID:${quarkus.uuid}}
```

And then add the following to our `Deployment` for the Fleet Manager:

```
env:
     - name: EVENT_BRIDGE_RESOURCES_WORKER_ID
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
```

If you want to have control over the worker id, set the `EVENT_BRIDGE_RESOURCES_WORKER_ID` environment variable. Otherwise let Quarkus generate a random UUID.

At that point I'm not event sure how much value the `WorkerIdProvider` brings over simply injecting the value of `event-bridge.resources.worker.id` directly into `WorkManagerImpl`.

WDYT?